cosmos/dbt/graph.py

```diff
@@ -52,6 +51,14 @@ class DbtNode:
     config: dict[str, Any] = field(default_factory=lambda: {})
     has_test: bool = False
 
+    @property
+    def name(self) -> str:
+        return self.unique_id.split(".", 2)[2].replace(".", "_")
+
+    @property
+    def select(self) -> str:
```

Comment:
I believe it can be confusing to name this property `select`, because of the selectors logic. Can you think about alternative names?

Same as before, it would be great to add a link to dbt documentation

Reply:
Perhaps, we could name this as `resource_name` or `defined_in`:
https://docs.getdbt.com/reference/resource-properties/versions#defined_in

I'm also happy for us to add support for `defined_in` in the future, depending on Cosmos users' needs, since this PR fixes a bug in Cosmos 1.2.

Reply:
`defined_in` is not in outputs of db ls.
```
DIVISION='abc' dbt ls --output json | grep '"resource_type": "model"'

{"name": "customers", "resource_type": "model", "package_name": "jaffle_shop", "original_file_path": "models/customers_v1.sql", "unique_id": "model.jaffle_shop.customers.v1", "alias": "customers_abc_v1", "config": {"enabled": true, "alias": "customers_abc_v1", "schema": null, "database": null, "tags": [], "meta": {}, "group": null, "materialized": "table", "incremental_strategy": null, "persist_docs": {}, "quoting": {}, "column_types": {}, "full_refresh": null, "unique_key": null, "on_schema_change": "ignore", "on_configuration_change": "apply", "grants": {}, "packages": [], "docs": {"show": true, "node_color": null}, "contract": {"enforced": false}, "post-hook": [], "pre-hook": []}, "tags": [], "depends_on": {"macros": [], "nodes": ["model.jaffle_shop.stg_customers", "model.jaffle_shop.stg_orders", "model.jaffle_shop.stg_payments"]}}
{"name": "customers", "resource_type": "model", "package_name": "jaffle_shop", "original_file_path": "models/customers.sql", "unique_id": "model.jaffle_shop.customers.v2", "alias": "customers_abc_v2", "config": {"enabled": true, "alias": "customers_abc_v2", "schema": null, "database": null, "tags": [], "meta": {}, "group": null, "materialized": "table", "incremental_strategy": null, "persist_docs": {}, "quoting": {}, "column_types": {}, "full_refresh": null, "unique_key": null, "on_schema_change": "ignore", "on_configuration_change": "apply", "grants": {}, "packages": [], "docs": {"show": true, "node_color": null}, "contract": {"enforced": false}, "post-hook": [], "pre-hook": []}, "tags": [], "depends_on": {"macros": [], "nodes": ["model.jaffle_shop.stg_customers", "model.jaffle_shop.stg_orders", "model.jaffle_shop.stg_payments"]}}
```

Reply:
To support this, it will require a bit more work, including parsing the manifest generated during `dbt_ls` - and we'd have additional work to support the custom load method.

I've logged a ticket so we don't lose track of this discussion:
https://github.com/astronomer/astronomer-cosmos/issues/685